src/assets/css/books.module.scss

```diff
@@ -0,0 +1,275 @@
+@import "./variables.scss";
+
+body {
+  width: 100%;
+  font-family: $RobotoSlab, $Monserrat;
+  color: $black;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+p {
+  font-size: small;
+}
+
+.blue {
+  color: $blue;
+  font-weight: 300;
+  font-size: small;
+  margin-bottom: 5%;
+}
+
+.img {
+  width: auto;
```

Comment:
- [ ] Hey good job you did a great job on this milestone stone but there are some issues you still need to address 
- [ ] I would suggest you stick to the original given design, your Styling should be  EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/), kindly  style the nav bar to fit the given design 
- [ ] see video screensrec below 

https://github.com/diarisdiakite/bookstore/assets/92458236/3da54b4f-9a37-47d4-8d03-89cb216ede1d

- [ ] given design
![bookstore](https://github.com/diarisdiakite/bookstore/assets/92458236/c189f64f-c354-4b64-8823-b8c241bbad26)